npm/src/db/dynamoDb.ts

```diff
@@ -330,6 +330,10 @@ class DynamoDB implements DatabaseDriver {
       );
     }
   }
+
+  async deleteMany(): Promise<void> {
+    throw new Error('Not implemented');
```

Comment:
For now just iterate the keys and call delete on each

Reply:
Noted.